### PR TITLE
Apply controller syntax from style guide

### LINKED ISSUE #26: Controllers should follow this style guide
Related #23 

Reference: https://github.com/johnpapa/angular-styleguide#controlleras-controller-syntax

### PR DESCRIPTION
**Pull request by alvarowolfx**
Closes #26

### CODE REVIEW

File: src/states/sidemenu/controller.js [hunk 1]
```diff
@@ -4,43 +4,40 @@
...
-    $scope.loginData = {};
+    vm.loginData = {};
 
     // Create the login modal that we will use later
     $ionicModal.fromTemplateUrl('modals/login.html', {
       scope: $scope
     }).then(function(modal) {
-      $scope.modal = modal;
+      vm.modal = modal;
+      $scope.vm = vm;
```
Comment (alvarowolfx) [inline src/states/sidemenu/controller.js]: I didn't liked this solution to pass ´´´$scope´´´ to the modal using the ´´´viewModel´´´ pattern, any good idea ? Should I just pass ´´´$scope´´´ in this case ?

Comment (felquis) [inline src/states/sidemenu/controller.js]: Me neither, I think we should defined it in the route definition, there's a option called controllerAs, but I couldn't make it work here. 

Do you have any ideia what is the solution?

Comment (alvarowolfx) [inline src/states/sidemenu/controller.js]: The ionicModal don't have the controllerAs option so I made this. It seems that  is creating a cycle on scope and view Model, I'll test this better

And I think I'll post a issue on Ionic to open the controllerAs/scopeAs possibility.

**Comment by felquis** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/states/sidemenu/controller.js (hunk 1 above)
Comment (felquis) [inline src/states/sidemenu/controller.js]: Could you please create another issue to discuss the usage of modals?

**Comment by felquis** [comment]
Closed by https://github.com/IonicBrazil/ionic-garden/commit/9be7081ab02cdbf2402935af3db75ce025235fd5

**Comment by felquis** [comment]
Thank you @alvarowolfx :)

**Pull request closed without merging**